tests/tier2/tc_2005_validate_password_option_by_cli.py

```diff
@@ -52,18 +52,26 @@ def test_run(self):
             results.setdefault('step2', []).append(res2)
 
         logger.info(">>>step3: password option is 红帽€467aa value")
+        pkg = self.pkg_check(self.ssh_host(), 'python-requests').split('-')[2]
         cli = self.vw_cli_base_update(base_cli,
                                       "--{0}-password=.*".format(hypervisor_type),
                                       "--{0}-password=红帽€467aa".format(hypervisor_type))
         data, tty_output, rhsm_output = self.vw_start(cli)
-        if "libvirt" in hypervisor_type:
-            logger.warning("libvirt-remote can use sshkey to connect, "
-                           "password value is not necessary")
-            res1 = self.op_normal_value(data, exp_error=0, exp_thread=1, exp_send=1)
-            results.setdefault('step2', []).append(res1)
+        if pkg[16:21] >= '2.20':
```

Comment:
It seems `python-requests` becomes one part of test case, is it possible to update python-requests to fix this issue? I mean if this is a test environment issue, we should not change our test steps.

Reply:
Removed checking python-requests pkg from cases and just use rhel build (rhel7 and rhel8) as judgment value because no such issue for rhel8, if the pkg is upgraded >=2.20 for rhel7 we need to test and change the cases based on new behaviors.